readme.txt

```diff
@@ -154,6 +154,18 @@ Many thanks go out to the fine folks who have helped us translate Pods into othe
  Join us in further translating the Pods interface at: http://wp-translate.org/projects/pods
 
 == Changelog ==
+= 2.5.1 - January, 22 2015 =
```

Comment:
`= 2.5.1 - January 22nd, 2015 =`

and 2.5 should be:

`= 2.5 - December 30th, 2014 =`